docker-compose.yml

```diff
@@ -0,0 +1,28 @@
+version: '3'
+services:
+  double_entry:
+    build: .
+    environment:
+      POSTGRES_HOST: postgres
+      MYSQL_HOST: mysql
+    volumes:
+      - ./lib:/double_entry/lib
+      - ./spec/double_entry:/double_entry/spec/double_entry
+      - ./spec/double_entry_spec.rb:/double_entry/spec/double_entry_spec.rb
+      - ./spec/generators:/double_entry/spec/generators
+      - ./spec/active_record:/double_entry/spec/active_record
+      - ./spec/performance:/double_entry/spec/performance
```

Comment:
I've avoided mounting the RSpec configuration, as we need a specific `database.yml` configuration in the Docker setup. I don't want to clobber anyone's local configuration.